mne/preprocessing/tests/test_scalp_coupling_index.py

```diff
@@ -0,0 +1,64 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+import pytest
+import numpy as np
+
+from mne.datasets.testing import data_path
+from mne.io import read_raw_nirx
+from mne.preprocessing.nirs import optical_density, scalp_coupling_index
+from mne.datasets import testing
+
+fname_nirx_15_0 = op.join(data_path(download=False),
+                          'NIRx', 'nirx_15_0_recording')
+fname_nirx_15_2 = op.join(data_path(download=False),
+                          'NIRx', 'nirx_15_2_recording')
+fname_nirx_15_2_short = op.join(data_path(download=False),
+                                'NIRx', 'nirx_15_2_recording_w_short')
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize('fname', ([fname_nirx_15_2_short, fname_nirx_15_2,
+                                    fname_nirx_15_0]))
+@pytest.mark.parametrize('fmt', ('nirx', 'fif'))
+def test_scalp_coupling_index(fname, fmt, tmpdir):
+    """Test converting NIRX files."""
+    assert fmt in ('nirx', 'fif')
+    raw = read_raw_nirx(fname)
+    raw = optical_density(raw)
+    sci = scalp_coupling_index(raw)
+
+    # All values should be between -1 and +1
+    assert (sci < 1.0).all()
+    assert (sci > -1.0).all()
```

Comment:
If you also know your data are good enough (whatever that means), I would add a:
```
assert_array_greater(np.abs(sci), 0.8)  # or whatever value is appropriate
```
The `np.corrcoef` should guarantee that we get -1 <= c <= 1, the computation doing something meaningful given our specific data quality should give us this > 0.8 (or whatever).

And maybe you can also add `assert_array_less(np.abs(sci), 0.98)` or something else appropriate?

Reply:
So this data isn't good. This short testing data was recorded on a manikin, so won't have a cardiac signal. I could do this test on the tutorial data, but then the tests will run a lot slower. Thats why I created the fake signals below. So can I just leave this as is?